server/service/src/invoice/outbound_shipment/update/mod.rs

```diff
@@ -411,5 +484,61 @@ mod test {
                 u
             })
         );
+
+        // helpers to compare totals
+        let stock_lines_for_invoice_lines = |invoice_lines: &Vec<InvoiceLineRow>| {
```

Comment:
Moved from graphql -> outbound updates 🤔 
Should I delete it from there since it's testing logic?

Reply:
I think 'yes'.. but don't listen to me. @clemens-msupply?

Reply:
@andreievg remove?

Reply:
Yes please